lib/resources/kernel_module.rb

```diff
@@ -37,13 +66,56 @@ def loaded?
       !found.nil?
     end
 
+    # @note maintain logcal agreement with 'disabled?' method
+    alias enabled? loaded?
+
+    def blacklisted?
+      if inspec.os.redhat? || inspec.os.name == 'fedora'
+        modprobe_cmd = "/sbin/modprobe --showconfig | grep blacklist | grep #{@module}"
+      else
+        modprobe_cmd = "modprobe --showconfig | grep blacklist | grep #{@module}"
+      end
+      cmd = inspec.command(modprobe_cmd)
+      cmd.exit_status.zero? ? cmd.stdout.delete("\n") : nil
```

Comment:
Sure, I can drop the `?`. I can see that convention? Perhaps with the `*_via_bin_{true|false}` methods should actually return a true or false or even perhaps all of them.

Does it make more sense to change the `cmd.stdout.delete("\n")` lines to return `true` rather than the standard out? Logically inspec would interrupt the `should be_*` and `should_not be_*` functions as a true/false value anyway correct? ( @arlimus I am thinking correctly on this? ) 

What do you think?